docs/architecture/adr-053-state-sync-prototype.md

```diff
@@ -0,0 +1,196 @@
+# ADR 053: State Sync Prototype
+
+This ADR describes an initial state sync prototype, and is subject to change as we gain feedback and experience.
+
+## Changelog
+
+* 2020-01-28: Initial draft (Erik Grinaker)
+
+## Context
+
+State sync will allow a new node to receive a snapshot of the application state without downloading blocks or going through consensus. This bootstraps the node significantly faster than the current fast sync system, which replays all historical blocks.
```

Comment:
> State sync will allow a new node to receive a snapshot of the application state without downloading blocks or going through consensus.

I assume this is how it's done now, right? It'd be cool to outline this and the current problems in a separate paragraph.

Reply:
This is outlined in ADR-042, which has a ton of background material on why this solution is chosen. This is mentioned below, but I'll add a line on it at the beginning.